magprime/automated_emails.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, event):
     SeasonSupporterEmail(_event)
 
 
-AutomatedEmail(Attendee, 'MAGFest schedule, maps, and other FAQs', 'precon_faqs.html', lambda a: days_before(7, c.EPOCH))
+AutomatedEmail(Attendee, 'MAGFest schedule, maps, and other FAQs', 'precon_faqs.html', lambda a: days_before(7, c.EPOCH) and a.badge_status not in [c.INVALID_STATUS, c.DEFERRED_STATUS] and a.paid is not c.NOT_PAID)
```

Comment:
``a.paid is not c.NOT_PAID`` is wrong, it needs to be ``a.paid != c.NOT_PAID``

We also need to handle badges in unpaid groups, so we should really be checking for whether a group has paid.

I'm planning on pushing out a fix later today for this.